solana-programs/audius_eth_registry/src/processor.rs

```diff
@@ -132,8 +329,267 @@ impl Processor {
             .map_err(|e| e.into())
     }
 
+    /// Process [ValidateMultipleSignaturesClearValidSigner]().
+    pub fn process_multiple_signatures_clear_valid_signer(
+        _program_id: &Pubkey,
+        accounts: &[AccountInfo],
+        signature_data_1: SignatureData,
+        signature_data_2: SignatureData,
+        signature_data_3: SignatureData,
+    ) -> ProgramResult {
+        let account_info_iter = &mut accounts.iter();
+        // initialized valid signer account 1
+        let valid_signer_1_info = next_account_info(account_info_iter)?;
+        // initialized valid signer account 2
+        let valid_signer_2_info = next_account_info(account_info_iter)?;
+        // initialized valid signer account 3
+        let valid_signer_3_info = next_account_info(account_info_iter)?;
+        // signer group account
+        let signer_group_info = next_account_info(account_info_iter)?;
+        // incoming valid signer account
+        let old_valid_signer_info = next_account_info(account_info_iter)?;
+        // Sysvar Instruction account info
+        let instruction_info = next_account_info(account_info_iter)?;
+
+        // Confirm program ownership of SignerGroup and ValidSigners
+        if valid_signer_1_info.owner != _program_id
+            || valid_signer_2_info.owner != _program_id
+            || valid_signer_3_info.owner != _program_id
+            || old_valid_signer_info.owner != _program_id
+            || signer_group_info.owner != _program_id {
+            return Err(AudiusError::InvalidInstruction.into());
+        }
+
+        // clock sysvar account
+        let clock_account_info = next_account_info(account_info_iter)?;
+        let clock = Clock::from_account_info(&clock_account_info)?;
+
+        // Recover all secp instructions present in this tx
+        let instruction_recovery = Self::recover_secp_instructions(&instruction_info);
+        if instruction_recovery.is_err() {
+            return Err(AudiusError::Secp256InstructionLosing.into());
+        }
+
+        let recovered_instructions = instruction_recovery.unwrap();
```

Comment:
Use `?` instead of `unwrap` (map errors to program errors if necessary)

Reply:
in order to avoid this error, must do some particular handling
```
+ exit 0
   Compiling audius_eth_registry v0.1.0 (/Users/hareeshnagaraj/Development/audius-protocol2/solana-programs/audius_eth_registry)
error[E0277]: `?` couldn't convert the error to `error::AudiusError`
   --> src/processor.rs:136:14
    |
136 |             )?;
    |              ^ the trait `std::convert::From<solana_program::sanitize::SanitizeError>` is not implemented for `error::AudiusError`
    |
    = note: the question mark operation (`?`) implicitly performs a conversion on the error value using the `From` trait
    = note: required by `std::convert::From::from`
```

Reply:
https://github.com/solana-labs/solana-program-library/blob/6dada44f75d4ba120ff248b8cc7ab74819e0010b/token-swap/program/src/processor.rs#L286